web/src/features/map-controls/MapButton.tsx

```diff
@@ -1,5 +1,4 @@
-import * as Toggle from '@radix-ui/react-toggle';
```

Comment:
Should we move this toggle thing to the button component or is it fine to just get rid of it?

Reply:
the asDiv prop I implemented for the update prompt should fix this. I'll double check that it doesn't cause any issues first though.